scripts/refresh-kani-proptest.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+# This file checks that the proptest rlib and symtab are up to are up
+# to date compared to the kani binary. If not, `cargo kani` is called
+# on proptest to refresh the symtab and rlib. This script is not meant
+# to called manually, but rather from `scripts/kani` and
+# `scripts/cargo-kani`.
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+KANI_REPO_ROOT="$SCRIPT_DIR/.."
+
+PROPTEST_SYMTAB_PATH="$(find $KANI_REPO_ROOT/target -name '*symtab.json' | head -1)"
```

Comment:
This seems fragile.

Reply:
Todo: enforce the fact that it is a proptest symtab.

Reply:
Implemented in [dbcbd2a](https://github.com/model-checking/kani/pull/1435/commits/dbcbd2ab5736692b6446cfad65cb82d820166953)